scriptworker/github.py

```diff
@@ -113,6 +116,15 @@ def get_tag_hash(self, tag_name):
             revision = self.get_tag_hash(tag_name=revision)
 
         repo = self._github_repository.html_url
+        if any(
+            vcs_rule.get('require_secret')
+            for vcs_rule in context.config['trusted_vcs_rules']
+        ):
+            # This check uses unofficial API on github, which we can't easily
+            # check for private repos, assume its true in the private case.
+            log.info("has_commit_landed_on_repository() not implemented for private"
+                     "repositories, assume True")
+            return True
```

Comment:
We could assume that `git@` or `ssh://` imply `require_secret`, but I'm ok explicitly flagging it.